packages/design-system/src/styles/components/_Accordion.scss

```diff
@@ -18,7 +18,7 @@ ds-accordion-item:not(:first-child) {
 .ds-c-accordion__button {
   background-color: var(--accordion__background-color);
   border: 0;
-  border-radius: var(--accordion__border-radius);
```

Comment:
I'm trying to determine why this needs to be broken up like this and it probably boils down to a  Figma limitation... `border-radius` is a shorthand property that can receive multiple values and CSS vars are just placeholders for strings (essentially), so could we keep `--accordion__border-radius` as a token and in the theme, assign it the values you want?

Like for mgov, maybe it's `--accordion__border-radius: .25em`, but for hgov it's `--accordion__border-radius: .25em .5em 2px`. Can Figma support that string of ".25em .5em 2px"?